packages/remark-lint-no-undefined-references/index.js

```diff
@@ -61,6 +62,17 @@
  *   > Eliding a portion of a quoted passage […] is acceptable.
  *
  * @example
+ *   {"name": "ok-allow.md", "config": {"allow": ["a", {"source": "^pkg\\."}, "b"]}}
+ *
+ *   [foo][pkg.subpkg]
+ *
+ *   [bar][a]
+ *
+ *   [baz][b]
+ *
+ *   Matching is case-insensitive: [bar][PKG.SUBPKG]
+ *
```

Comment:
I think it’s better to have one example that fails here (`b`?)

And maybe instead of `pkg.subpkg` here, a value of `c.d` might make more sense? Conveys the same, while still using abstract values

Reply:
> I think it’s better to have one example that fails here (`b`?)

See 73a7b0e1d2829540843310848ee66080d3266d0b. WDYT?

> And maybe instead of `pkg.subpkg` here, a value of `c.d` might make more sense? Conveys the same, while still using abstract values

See ae0f12b00ceb516fbb649c8ab00741891a600d1a. WDYT?